changelogs/fragments/1641_extend_result_docs_guest_disk

```diff
@@ -0,0 +1,2 @@
+trivial:
```

Comment:
```suggestion
minor_changes:
```

To the best of my knowledge, `trivial` is not a valid changelog category.

Reply:
At least, as far as I understand `trivial` won't show up in the changelog. And I think this change should.